### ISSUE TITLE
javax to jakarta migration

### ISSUE THREAD

**Issue by MartinHaeusler** [pingidentity/scim2#179]
Hello everyone,

since the newly released Spring Boot 3 is now fully based on the new jakarta.* namespaces, it would be great if this package could be upgraded to use jakarta as well.

Thanks!

**Comment by Fabio1988** [comment]
#170 
#173

**Comment by kqarryzada** [comment]
We plan to address this issue within the next couple of months (ideally sooner, but I cannot make any promises about that). Implementing this change will involve extra uplift on our part, since Spring Boot doesn't just require Jakarta EE, it actually requires Jakarta EE 9 (citation: https://spring.io/blog/2021/09/02/a-java-17-and-jakarta-ee-9-baseline-for-spring-framework-6). Some time ago, I did some initial work to migrate our codebase to Jakarta EE 8, but I abandoned this when I realized that it was insufficient.

Thus, this change will require that the latest release of the SCIM SDK drops support for JDK 8. We are in the process of doing this, but we need to work out the logistics since some Ping Identity products still support this JDK.

Thanks for the continued interest in this!
